docs/source/Lines_functionality.ipynb

```diff
@@ -0,0 +1,1861 @@
+{
```

Comment:
<p>Can shorten this link too.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OSOceanAcoustics/echoregions/pull/184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>